streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/config/types/SlotAssignStrategyType.java

```diff
@@ -0,0 +1,24 @@
+package org.ray.streaming.runtime.config.types;
+
+/**
+ * Slot assign strategy type.
+ */
+public enum SlotAssignStrategyType {
+
+  /**
+   * PIPELINE_FIRST_STRATEGY
```

Comment:
same here. Please describe what does it mean when the PIPELINE_FIRST_STRATEGY is used.